.golangci.yml

```diff
@@ -21,13 +20,15 @@ linters:
     - nakedret
     - nolintlint
     - staticcheck
-    - structcheck
     - typecheck
     - unconvert
     - unparam
     - unused
-    - varcheck
 linters-settings:
+  revive:
+    rules:
+      - name: package-comments
+        disabled: true
```

Comment:
Ah, you found the cleaner way to disable it 😂

Reply:
I ❤️ the JSON schema support in IntelliJ for stuff like this!

(Also, I totally would have just copy-pasted from your PR but GitHub was erroring out on me at the time 🙈)